### ISSUE TITLE
Fine tuning

### ISSUE THREAD

**Issue by rishabh063** [plemeri/InSPyReNet#44]
Hey I have been fine tuning using pre-trained checkpoint . however Having issues for it to converge , I have 20k Mask and image set for background removal . I have run 30 Epochs and the results are worse than the pretrained one

Any advice on how many epoch or learning rate ?

**Comment by plemeri** [comment]
Finetuning with additional data only may cause catastrophic forgetting. Try adding your data to the composite dataset.

**Comment by LukaGiorgadze** [comment]
> Finetuning with additional data only may cause catastrophic forgetting. Try adding your data to the composite dataset. 

I'm just curious - what's the reason of that?

**Comment by rishabh063** [comment]
So you mean i Need to train on the pretraining dataset and my own dataset ?

**Comment by rishabh063** [comment]
@plemeri  I saw that in your transparent background libraries nightly model scores much higher than HR model in model zoo when tested on DUTS-TE . Can you share the reason and training technique ? is it just trained longer ?

**Comment by plemeri** [comment]
Nightly model was trained with the updated training strategy which was my doctorate dissertation. Currently I have no time to release updated contents.

**Comment by rishabh063** [comment]
@plemeri if you can share it

**Comment by plemeri** [comment]
My dissertation is not publicly opened yet because of the ongoing patent registration. Sorry for that.

**Comment by LukaGiorgadze** [comment]
@plemeri congrats on your doctorate dissertation! 👍 
I would be more than happy to see the updates on this repository and not kill as everyone does after some time. We are waiting updated training strategy at some point.

**Comment by MHillier98** [comment]
Could you point us in the direction of these better methods then? :)
